### ISSUE TITLE
Empty state action relationship error

### ISSUE THREAD

**Issue by damms005** [laravel-filament/filament#8785]
### Package

filament/filament

### Package Version

v3.0.62

### Laravel Version

v10.25.2

### Livewire Version

v3.0.5

### PHP Version

8.1.23

### Problem description

"Create Action" button in empty state does not populate the related column with the correct value. Details [here](https://github.com/filamentphp/filament/discussions/8771)

### Expected behavior

"Create Action" button in empty state should populate the related column with the correct value.

### Steps to reproduce

- [ ] Clone https://github.com/damms005/example-app
- [ ] Checkout the `bug/relationship-fails`
- [ ] Setup database `php artisan migrate:fresh --seed`
- [ ] Start server `php artisan serve`
- [ ] Navigate to http://127.0.0.1:8000/admin/1/players (or similar per your server URL)
- [ ] Click on the empty state "New player" button (not the page action "New player" button)
- [ ] Enter a level and create
- [ ] You should see notification that created successfully
- [ ] Your database should have a new Player with empty `team_id`
- [ ] Now use the page action "New player" button to create a new player
- [ ] Your database should have a second Player with correct `team_id`

### Reproduction repository

https://github.com/damms005/example-app/tree/bug/relationship-fails

**Comment by zepfietje** [comment]
Make sure all actions on the same page have a unique name.
In this case you should pass a name other than the default `create` name to either of the `CreateAction::make()` methods.

**Comment by damms005** [comment]
Do you accept a PR to add this detail to the docs?

This is because the action buttons are the default ones out-of-the-box, so I wrongly thought such nuances did not matter.

**Comment by zepfietje** [comment]
> Do you accept a PR to add this detail to the docs?

Yeah I think we do, though I thought @danharrin resolved this recently.
Let's see what he thinks.

**Comment by damms005** [comment]
Also, I get this error when I tried to add the action name:

`Non-static method Filament\Actions\StaticAction::name() cannot be called statically`

![image](https://github.com/filamentphp/filament/assets/9839355/c01971f6-99c7-4d35-86f9-50b3fd4046f8)

**Comment by damms005** [comment]
@zepfietje Should this be a new issue?

**Comment by damms005** [comment]
IMO, with respect to the OP, it should either be documented (we still waiting on @danharrin for this), or it should error out (perhaps throw an exception) rather than silently enter wrong tenant info. Thoughts? @zepfietje

**Comment by zepfietje** [comment]
@damms005, you should pass the name to the `make()` method like I said above, so e.g. `CreateAction::make('emptyStateCreate)`.

**Comment by damms005** [comment]
Got it. Thank you Zep!
